packages/openapi-generator/src/parser/type-mapping.ts

```diff
@@ -26,6 +26,7 @@ const typeMapping = {
   map: 'any',
   date: 'string',
   DateTime: 'string',
+  Blob: 'Blob',
```

Comment:
[q] This is never used, is it?

Reply:
Yes, its not used after the below change.